Lib/pathlib.py

```diff
@@ -1361,13 +1361,23 @@ def lstat(self):
             self._raise_closed()
         return self._accessor.lstat(self)
 
-    def link_to(self, target):
+    def link_to(self, link):
         """
-        Create a hard link pointing to a path named target.
+        Make *link* a hard link pointing to the same file as this path.
+
+        Deprecated since Python 3.9, use `hardlink_to()`.
```

Comment:
Where is the `DeprecationWarning`? Also need to specify when removal is planned (either 3.11 or 4.0 depending on how bad it will be to keep it around).

Reply:
I'm not sure whether this would be removed in 3.11 or 4.0 - how can I work that out?

Reply:
@barneygale Have a discussion on the issue.